docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+1. For both FASTQ and BAM submission, all files in the `files` section must be unique in the payload.
+1. All `read_groups` in the payload and BAM header must belong to a single sample.
+1. `platform_units` must be unique with a one-to-one relationship with `submitter_read_group_id`.
+1. The total number of `read_group` objects must match the number specified in `read_group_count`.
+1. For paired-end sequencing, both `file_r1` and `file_r2` are required.  For single-end sequencing, only `file_r1` is required (`file_r2` must be populated with `null`).
+1. For paired-end sequencing, `file_r1` and `file_r2` must not be the same file. 
+1. For FASTQ submission, no file can appear more than once in `file_r1` or `file_r2` across read group objects.
+
+### Understanding the Payload Fields
+:::note   
+![Required](/assets/submission/dictionary-required.svg) indicates a field that **must** be included in the metadata payload or it will immediately fail submission.
+:::
+
+The metadata payload is broken down into 4 sections: `experiment`, `samples`, `read groups` and `files`, with all other fields being at the root level. Each section **must** be submitted in the payload.
+
+The `samples` section contains details of the clinical data and key sample descriptors related to the submitted files. In order to submit a metadata payload, samples must be [registered](/docs/submission/registering-samples) in the ARGO Data Platform. For permissible values of all fields, please see the Sample Registration file of the [Data Dictionary](/dictionary). If the data for a sample is different than what has been registered, metadata submission will fail.
+
+A description of all other payload sections is shown below: 
+
+|  Payload Field     | Payload Section |  Attribute                                                |  Description                                                                                                                                                                  |  Permissible Values     |  
+|-|-|-|-|-|
+|  **studyId**           |  |  ![Required](/assets/submission/dictionary-required.svg)  |  ARGO `Program ID`, the unique identifier of your program. If you have logged into the ARGO Data Platform, this is the Program ID that you see in the Program Services area.  |                         |  
+|  **analysisType**      |  |  ![Required](/assets/submission/dictionary-required.svg)  |  The type of molecular data that is being submitted in the payload.                                                                                                           |  **sequencing_experiment**  |  
+|  **submitter_sequencing**\_ **experiment_id**  | experiment |  ![Required](/assets/submission/dictionary-required.svg)  |  The unique identifier of the sequencing experiment**.                                                     |                                    |  
```

Comment:
For field `Permissible values`, do we want to add the following according to the schema:
```
String values must meet the regular expression:
^[a-zA-Z0-9]{1}[a-zA-Z0-9\\-_\\.:']{0,98}[a-zA-Z0-9]{1}$
Or `null`
```